### DISCUSSION TITLE
How can I tell which version I'm currently using ?

### DISCUSSION THREAD

**Discussion by SteveBrailsord** [nathanmarlor/foxess_em#210] [Q&A]
how can I tell which version I'm currently using and how do I update - installed through HACs
I dont seem to have the solcast fix for API count ?

**Comment by nathanmarlor** [comment]
Version number within HACS -> 
![image](https://user-images.githubusercontent.com/3436911/225625961-49f7df5f-7d41-4d51-8995-4e3b7baed022.png)

**Comment by SteveBrailsord** [comment]
Thank you
Feel such a numpty 😂
